### PR TITLE
Add Spamhaus DBL analyzer

### PR DESCRIPTION
**Pull request by weslambert**
Related to https://github.com/TheHive-Project/Cortex-Analyzers/issues/436

### CODE REVIEW

**Comment by nadouani** [comment]
Hello @weslambert Thanks for the PR, is it possible to add a PR for https://github.com/TheHive-Project/CortexDocs/blob/master/analyzer_requirements.md

**Comment by weslambert** [comment]
PR submitted for docs: https://github.com/TheHive-Project/CortexDocs/pull/56

File: analyzers/SpamhausDBL/SpamhausDBL.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "name": "SpamhausDBL",
+    "version": "1.0",
+    "author": "Wes Lambert",
+    "url": "https://github.com/TheHive-Project/Cortex-Analyzers",
+    "license": "AGPL-V3",
+    "description": "Perform domain lookup to Spamhaus DBL",
+    "dataTypeList": ["domain"],
```
Comment (nadouani) [inline analyzers/SpamhausDBL/SpamhausDBL.json]: Is it meaningful to add `fqdn` to that list? I don't know SpamhausDBL.

Comment (weslambert) [reply analyzers/SpamhausDBL/SpamhausDBL.json]: Certainly, I'll look into that. 👍

Comment (weslambert) [reply analyzers/SpamhausDBL/SpamhausDBL.json]: `fqdn` has been added ➕.

**Pull request merged**